src/client/components/intro/Intro.js

```diff
@@ -9,6 +9,20 @@ const Intro = () => {
   const [name, setName] = useState('');
   const [task, setTask] = useState('');
   const [bears, setBears] = useState();
+  const words = [
```

Comment:
This can still be moved outside of the component function, to the top of the file.